packages/components/addon/components/hds/code-block/index.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { assert } from '@ember/debug';
+import { htmlSafe } from '@ember/template';
+import { guidFor } from '@ember/object/internals';
+import Prism from 'prismjs';
```

Comment:
```suggestion

import Prism from 'prismjs';
```

Reply:
Updated in 87a30a9fcfd749cde5ab91234bdb3bd5b3526e89